app/mailers/ch31_submissions_report_mailer.rb

```diff
@@ -3,6 +3,8 @@
 require 'reports/uploader'
 
 class Ch31SubmissionsReportMailer < ApplicationMailer
+  include BooleanHelper
```

Comment:
This doesn't need to be here anymore, right?

Reply:
Yeah, CI just caught it, and it's gone.